templates/echo.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: demo-deployment 
+spec:
+  replicas: 3 
+  template:
+    metadata:
+      labels:
+        app: demo-app 
+    spec:
+      containers:
+      - name: echoheaders 
+        image: gcr.io/google_containers/echoserver:1.4

```

Comment:
Ditto here

Reply:
Yeah, I couldn't find an official in dockerhub so went with this one. Let me poke around and try to find one.

Reply:
could you just pull the latest and tag it and push to hub? If you want I can make you a repo under `awspartners` for Docker Hub?